flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -330,8 +325,10 @@ static int32_t vtol_follower_control_simple(const float dT,
 
 	/* Calculate the difference between where we want to be and the
 	 * above position */
-	float horiz_error_norm = vtol_calculate_distances(cur_pos_ned,
-		hold_pos_ned, errors_ned, true);
+	vtol_calculate_distances(cur_pos_ned, hold_pos_ned, errors_ned, 0);
+
+	float horiz_error_mag = vtol_magnitude(errors_ned, 2);
```

Comment:
Basically, the change you had me make was to take the difference, calculate the norm and normalize, before scaling back up with a deadband on the norm.

My code instead takes the difference, calculates the magnitude of the first 2 elements, and adjusts the deadband.  The Z component should not be involved in normalization of the vector or the deadband.